website/content/security.mdx

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Have switched over to the `markdown-page` approach to `/security` to match other docs properties.